url.bs

```diff
@@ -844,8 +844,9 @@ resource the <a for=url>URL</a>'s other components identify. It is initially nul
 
 <p class="note no-backref">This is not an <a>ASCII string</a> on purpose.
 
-<p>A <a for=url>URL</a> also has an associated <dfn>non-relative flag</dfn>. It is
-initially unset.
+<p>A <a for=url>URL</a> also has an associated
+<dfn><span id=non-relative-flag>cannot-be-a-base-URL flag</span></dfn>.
```

Comment:
The `<span>` is to keep external links working. Bikshed has an `oldids` attribute for this, but (for now) it only applies to headings.